File: DemoMMOUnity/Lua/Game/GenerateScripts/UIModules/BagUIPanelView.lua
---@class BagUIPanelView
BagUIPanelView = class("BagUIPanelView")
function BagUIPanelView:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.equipmentOtem = self._UIView:GetObjTypeStr("equipmentOtem") or CS.UnityEngine.GameObject
end

File: UnityMMOClient/Assets/Game/Hoftix/Script/GenerateScripts/UIModules/BagUIPanelView.cs
using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class BagUIPanelView:UIViewInterface
    {
        public UnityEngine.GameObject equipmentOtem=null;
		


        public void Init(UIView _view)
        {
            equipmentOtem=_view.GetObjType<UnityEngine.GameObject>("equipmentOtem");
			
        }
    }
}